### PR TITLE
Shij10 adds Chengdu, China

### LINKED ISSUE #56: shij10 follows Github Flow
Chengdu, China

- [x]  create a **branch**
- [x]  add some **commits**
- [x]  open a *pull request*
- [ ]  collaborate
- [ ]  `merge` with :+1:

### PR DESCRIPTION
**Pull request by shij10**
This pull request is to add Chengdu, China to the list of hometowns

@crichID welcome to my city.

The PR closes #56

### CODE REVIEW

**Comment by crichID** [comment]
> welcome to my city.

@shij10 thank you! I look forward to visiting.

File: shij10-chengdu.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Changedu, China
```
Comment (Damaris-Landers) [inline shij10-chengdu.md]: Wow! I would love to visit China someday.

Comment (Damaris-Landers) [inline shij10-chengdu.md]: Where is People Park?

Comment (shij10) [inline shij10-chengdu.md]: it is done in 9050f831c34354e23b037b418f7b6fa69ed14a7d

**Pull request merged**